app/api/payments/paypal/capture/route.ts

```
@@ -46,40 +52,43 @@ export async function POST(request: NextRequest) {
       }
 
       // Send notification to user
       const { data: transaction } = await supabase
         .from("transactions")
         .select("user_id, amount")
         .eq("reference", orderId)
-        .single()
+        .single();
 
-      if (transaction) {
+      if (transaction && transaction.user_id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)